x-pack/plugins/security_solution/server/lib/telemetry/index.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { UsageCollectionSetup } from 'src/plugins/usage_collection/server';
+import { ISavedObjectsRepository } from '../../../../../../src/core/server';
+import { getEndpointTelemetryFromFleet, EndpointUsage } from './endpoint';
+
+export interface UsageData {
+  endpoint?: EndpointUsage;
+}
+
+export async function createSiemTelemetry(
+  usageCollector: UsageCollectionSetup,
+  savedObjectsClient: ISavedObjectsRepository
+) {
+  const collector = usageCollector.makeUsageCollector<UsageData>({
+    type: 'security_solution',
```

Comment:
Q: Bear in mind this collector is registering with the same `type` as the one specified in https://github.com/elastic/kibana/pull/71102. Whoever merges first will take ownership of it and it will break the other's implementation

cc/ @rylnd

Reply:
Yep! Plan is for him to merge his first and then I'll make the updates for mine on top of his changes once they're in